### PR TITLE
fix: added glass effect

### PR DESCRIPTION
**Pull request by bhattabhi013**
What:
Added glass effect behind the main content.
It fixes issue #37.

Screenshot:
image

### CODE REVIEW

**Comment by imaaquibali** [comment]
@bhattabhi013 Resolve the conflict!

**Comment by bhattabhi013** [comment]
Hi @imaaquibali,
Resolved the conflict, please merge.

File: style.css [hunk 1]
```diff
@@ -389,4 +389,16 @@ footer #email:hover {
     width: 70px;
   }
 
+}
+.container-glass {
```
Comment (IAmTamal) [inline style.css:393]: We can also add a small `Border radius` - it'll give a premium look.
Plus, your branch still has conflicts , use `git pull origin <origin branch name>` to pull and resolve all the conflicts.

**Review by IAmTamal — CHANGES_REQUESTED** [review]
your branch still has conflicts , use `git pull origin <origin branch name>` to pull and resolve all the conflicts.

File: style.css (hunk 1 above)
Comment (bhattabhi013) [reply style.css]: Changes done.
image

**Comment by bhattabhi013** [comment]
> your branch still has conflicts , use `git pull origin <origin branch name>` to pull and resolve all the conflicts.

Resolved the issues now, please check.

**Review by Susmita-Dey — COMMENTED** [review]
Doesn't look good to me. Closing this for now

**Pull request closed without merging**